metabase/metabase
# Mantine Card

Epic https://github.com/metabase/metabase/issues/32268
Context: https://metaboat.slack.com/archives/C057WD5L0JG/p1693935868971129

How to verify:
- `yarn storybook`
- Open http://localhost:6006/?path=/docs/data-display-card--default-story
- Make sure `Paper` properties can be configured (`radius` etc)

<img width="1342" alt="Screenshot 2023-09-26 at 10 28 46" src="https://github.com/metabase/metabase/assets/8542534/6426b675-8782-40d3-b8cc-8d1a4efd8995">
